stages/test/test_skopeo.py

```diff
@@ -118,3 +118,28 @@ def test_skopeo_copy_oci(tmp_path, stage_module):
     data_digest = data["manifests"][0]["digest"].split(':')
     manifest_file = result_path / "blobs" / data_digest[0] / data_digest[1]
     assert_manifest_file(manifest_file)
+
+
+@pytest.mark.parametrize("remove_signatures", [True, False])
+def test_skopeo_copy_remove_signatures(tmp_path, stage_module, remove_signatures):
+    # We are completely faking everything here, because we mock out skopeo
+    fake_oci_path = tmp_path / "fake-container"
+    inputs = make_fake_images_inputs(fake_oci_path, "some-name")
+
+    with testutil.mock_command("skopeo", "") as args:
+        options = {
+            "destination": {
+                "type": "oci",
+                "path": "/tmp/test-output-skopeo-dir",
+            },
+            "remove-signatures": remove_signatures,
+        }
+        output_dir = tmp_path / "output"
+        stage_module.main(inputs, output_dir, options)
+
+        if remove_signatures:
+            # Check that skopeo has --remove-signatures right after the copy subcommand
```

Comment:
(super nitpick) I personally would leave the two comments out, they are examples of "what" comments, i.e. explaining what the code is doing but (at least to me) the code feels nice and clean and straightforward so reading the code should be enough (also less risk this way of drift between comments and code). In general I try to avoid "what" comments because they tend to indicate the code could be cleaner in which case they are not needed :)

(side-note: I personally like the "why" comments  better, i.e. explaining "why" something is done the way it's done but here again it feels not necessary).

*But* this of course is a super nitpick and very subjective/personal, I really like that there are comments and that this is nice and well written, don't take it the wrong way :)